urbansim/developer/sqftproforma.py

```diff
@@ -378,6 +378,9 @@ def _generate_lookup(self):
                     df['park_sqft'] = 0
                     # not all fars support surface parking
                     stories[np.where(stories < 0.0)] = np.nan
+                    # I think we can assume that stories over 3
+                    # do not work with surface parking
+                    stories[np.where(stories > 3.0)] = np.nan
```

Comment:
The `np.where` isn't necessary here, I think `stories[stories > 3.0]` would work.